src/pages/SingleVideo/SingleVideo.css

```diff
@@ -0,0 +1,48 @@
+.single-video-container {
+	background-color: var(--bg-color);
+	width: 83%;
+	min-height: 100vh;
+	margin: 0 auto;
```

Comment:
The height of the react player itself is very low.
You can add ```aspect-ratio : 1/1``` to make it equal on both sides.